src/kOS/Function/BuildList.cs

```diff
@@ -40,7 +40,7 @@ public override void Execute(SharedObjects shared)
                     list = shared.Vessel.PartList(listType, shared);
                     break;
                 case "files":
-                    list = ListValue.CreateList(shared.VolumeMgr.CurrentVolume.GetFileList());
+                    list = ListValue.CreateList(shared.VolumeMgr.CurrentVolume.FileList.ToList());
```

Comment:
I'm getting an encapsulation exception when running `list files in filelist.`.  It's complaining about a key value pair, so I suspect that this line is supposed to be `FileList.Values.ToList());` which should only put the `VolumeFile`s themselves into the list.  I will test locally to confirm.